src/ebay-dialog-base/components/dialogBase.tsx

```diff
@@ -16,15 +16,13 @@ import * as keyboardTrap from 'makeup-keyboard-trap'
 import { EbayIcon } from '../../ebay-icon'
 import { randomId } from '../../common/random-id'
 import { useDialogAnimation, TransitionElement } from './animation'
+import { ClassPrefix } from '../types'
+import EbayDialogFooter from './dialog-footer'
 import { DialogCloseEventHandler } from '../types'
 
-
 export type WindowType = 'compact'
-type ClassPrefix = 'fullscreen-dialog' | 'lightbox-dialog' | 'panel-dialog'
-    | 'drawer-dialog' | 'toast-dialog' | 'alert-dialog' | 'confirm-dialog'
-    | 'snackbar-dialog'
-type ButtonPosition = 'top' | 'right' | 'bottom' | 'left' | 'hidden'
 
+type ButtonPosition = 'right' | 'left' | 'hidden'
```

Comment:
I think the elimination of `bottom` here is desirable & does not require a major version bump. It is not used inside this codebase, and it appears that the `buttonPosition` prop is _not_ exposed by any of the wrapping dialog components. Since `EbayDialogBase` not available to consumers, it is not a breaking change.